app/components/jurisdiction-lists.js

```diff
@@ -34,7 +34,14 @@ export default Ember.Component.extend({
         function(_data) {
           this.attrs.selectJurisdiction(_data.data.id, _data.data.title)
         }.bind(this),
-        function(err) {}
+        function(err) {
+          swal({
+            type: "error",
+            title: "Oh no!",
+            text: "Go find Scott or Marika",
```

Comment:
when writing errors, always assume someone that's not Cc will see this. So, in this case, change it to something like, "Contact support@commoncurriculum.com" and change the title to, "We've run into an error creating the jurisidction" or something like that.